src/components/cc-env-var-create/cc-env-var-create.js

```diff
@@ -20,7 +20,7 @@ import { linkStyles } from '../../templates/cc-link/cc-link.js';
  * * The validation of the variable name format is handled with [@clevercloud/client](https://github.com/CleverCloud/clever-client.js/blob/master/esm/utils/env-vars.js)
  * * The validation of existing names is handled with the `variablesNames` property which is a list of already existing names.
  *
- * @cssdisplay block
+ * @cssdisplay flex
```

Comment:
I think the layout is broken for `650px` and `450px` viewport thresholds (inputs and button do not collapse anymore).

Reply:
ah yes, this is part of a quick test following Hubert's feedback, shouldn't have been pushed like this.
Good catch, thanks :hugs: